api/RTCPeerConnection.json

```diff
@@ -0,0 +1,1987 @@
+{
+  "api": {
+    "RTCPeerConnection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection",
+        "support": {
+          "webview_android": [
+            {
+              "notes": "<code>-webkit</code>",
```

Comment:
This needs to use the "prefix" property: 
https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#prefix
"prefix": "webkit" (no dash)

The prefix is not needed on the sub features of RTCPeerConnection.

If you could move the RTCPeerConnection.json file to a new separate PR, it would be better for reviewing maybe. It is quite giant.

Reply:
Yes, apologies. On it.